packages/nestjs/jest.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../jest/jest.config.js');
```

Comment:
Hm, wondering if we shouldn't use `vitest` instead. 
cc @mydea @lforst

Reply:
Yeah I like that. @nicohrubec would you mind adding a vitest setup instead of jest. You can take inspiration from the  `svelte`, `sveltekit` and `solid` packages.